docs/content/support/v16-migration.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Primer v16 Migration Guide
```

Comment:
What's the URL for this file? I still find myself needing to refer to the new names like `color-text-secondary` instead of `text-gray`, and the old URL I was using, https://github.com/primer/css/blob/mkt/color-modes-docs/docs/content/support/v16-migration.mdx#utility-classes, now 404s. 🙇‍♀️

Reply:
@cheshire137 Eventually it will show up in https://primer.style/css, but for now you can see it in this preview: https://primer-css-git-release-1600-primer.vercel.app/css/support/v16-migration

Reply:
Thank you!